src/overlays/actors/ovl_En_Zov/z_en_zov.c

```diff
@@ -360,7 +360,7 @@ s32 func_80BD1AE0(EnZov* this, PlayState* play) {
 
 void func_80BD1BF0(EnZov* this, PlayState* play) {
     if (!func_80BD1AE0(this, play)) {
-        func_80BD1570(this, 0, 0);
+        func_80BD1570(this, 0, ANIMMODE_LOOP);
```

Comment:
If you're naming all this stuff, could you name the functions using the animations too?

Reply:
I think all the naming would be better off on a full documentation pass for these undocumented actors. Especially since some of these actors have wrappers for their general change animation function.